ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -171,21 +171,21 @@
                       or limited partnership. The owners of the proprietorship or partnership must be added as a
                       person or as an organization.
                     </p>
-                    <hr class="mb-5 mt-6 solid"/>
+                    <hr class="mb-5 mt-6 solid">
                     <h3 class="text-center mb-2">
                       When B.C. Based Businesses Must be in Active Status
                     </h3>
                     <p><b>New owners:</b> Must be active at the time of registration.</p>
                     <p>
                       If you are adding a B.C. based business as a new owner, the business <b>must be active on the
-                      B.C Corporate Register at the time of the registration.</b>
+                        B.C Corporate Register at the time of the registration.</b>
```

Comment:
Should this be "B.C." instead of "B.C"?

Reply:
Indeed, I imagine it should, good eye, good thing this line was linted 😁 
Thank you!